client/src/components/SudoModePasswordField/SudoModePasswordField.js

```diff
@@ -15,14 +15,41 @@ function SudoModePasswordField(props) {
   const {
     onSuccess,
     autocomplete,
+    collapsed,
     verifyMessage,
+    sectionTitle,
   } = props;
   const passwordFieldRef = createRef();
   const [responseMessage, setResponseMessage] = useState('');
   const [showVerify, setShowVerify] = useState(false);
+  // setting to null initially to prevent a fout as useEffect()
+  // below will cause a re-render when collapsed
+  const [expanded, setExpanded] = useState(null);
+
+  useEffect(() => {
+    setExpanded(!collapsed);
+  }, [collapsed]);
```

Comment:
[Screencast from 12-03-25 17:37:35.webm](https://github.com/user-attachments/assets/389cfbe8-cec3-4b3b-8a3e-9b8e5a66c212)

This is what it looks like for me when I make the following change:
```suggestion
  const [expanded, setExpanded] = useState(!collapsed);
```

There is the barest almost-imperceptible moment while the field isn't visible - I think that's fine. It would only be a problem if it was rendering something else first, which it isn't.

Reply:
OK, looked a lot worse on my local.  I believe it's also necessary for storybook so that you can use the controls to toggle between collapsed and not collapsed, so will leave as is